task3/server/server.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const fs = require('fs');
+const bodyParser = require('body-parser');
+const { v4: uuidv4 } = require('uuid');
+
+const port = 3000;
+
+const app = express();
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.listen(port, () => {
+  console.log('Server is running');
+});
+
+app.get('/users', (req, res) => {
+  fs.readFile('./task3/db/users.json', 'utf-8', (err, data) => {
+    if(err) console.log("error: ", err);
+    const users = JSON.parse(data);
+    res.send(users);
+  });
+});
+
+app.get('/users/:id', (req, res) => {
+  let { id } = req.params;
+  fs.readFile('./task3/db/users.json', 'utf-8', (err, data) => {
+    if(err) console.log("error: ", err);
+    const users = JSON.parse(data);
+    const user = users.filter(user => user.id === id);
```

Comment:
It's not correct to use filter here. Filter returns you array of items that match your condition, which means it will check EVERY element in your array. What if I need to get user with id 1 and there is million users, it will take eternity